media-libs/openal/openal-1.13.0.recipe

```diff
@@ -8,14 +8,15 @@ HOMEPAGE="https://www.openal-soft.org/"
 COPYRIGHT="1999-2000 Loki Software
 	2005-2011 OpenAL Soft team"
 LICENSE="GNU LGPL v2.1"
-REVISION="4"
-SOURCE_URI="https://kcat.strangesoft.net/openal-releases/openal-soft-1.13.tar.bz2"
-CHECKSUM_SHA256="dc735b8b2ab21f6ec54b4262f150a7e0527ae42aa975a1965b9342df1520443c"
-SOURCE_DIR="openal-soft-1.13"
-PATCHES="openal-1.13.0.patchset"
+REVISION="5"
+SOURCE_URI="https://github.com/kcat/openal-soft/archive/refs/tags/openal-soft-1.13.tar.gz"
+#SOURCE_URI="https://kcat.strangesoft.net/openal-releases/openal-soft-1.13.tar.bz2"
+CHECKSUM_SHA256="6f39476853c1b21ea841401b7f15bc3ed6b6c738b0a840de076a2865eb7751f9"
+SOURCE_DIR="openal-soft-openal-soft-1.13"
+PATCHES="openal-$portVersion.patchset"
 
-ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="!all x86_gcc2"
```

Comment:
You can just leave only x86_gcc2 and remove the others (i.e. no need for `!`, they don't have to be listed at all.)